developer-docs/bolt-server.md

```diff
@@ -22,6 +22,18 @@ Bolt server can be configured by defining content in HOCON format at one of the
 - `whitelist`: Array, *optional* - A list of hosts which can connect to pe-bolt-server.
 - `concurrency`: Integer, *optional* - The maximum number of server threads (default `100`).
 
+**Environmnet Variable Options**
+The following configuration options can be set with environment variables. 
+- `SSL_CERT`
```

Comment:
I feel like these should have a BOLT_ prefix to avoid accidentally being overridden.

Reply:
That is a great idea. Will revise.